proto/engine/v1/engine.ssz.go

```diff
@@ -3222,3 +3222,233 @@ func (b *BlobsBundle) HashTreeRootWith(hh *ssz.Hasher) (err error) {
 	hh.Merkleize(indx)
 	return
 }
+
+// MarshalSSZ ssz marshals the BlobsBundleV2 object
+func (b *BlobsBundleV2) MarshalSSZ() ([]byte, error) {
+	return ssz.MarshalSSZ(b)
+}
+
+// MarshalSSZTo ssz marshals the BlobsBundleV2 object to a target array
+func (b *BlobsBundleV2) MarshalSSZTo(buf []byte) (dst []byte, err error) {
+	dst = buf
+	offset := int(12)
+
+	// Offset (0) 'KzgCommitments'
+	dst = ssz.WriteOffset(dst, offset)
+	offset += len(b.KzgCommitments) * 48
+
+	// Offset (1) 'Proofs'
+	dst = ssz.WriteOffset(dst, offset)
+	offset += len(b.Proofs) * 48
+
+	// Offset (2) 'Blobs'
+	dst = ssz.WriteOffset(dst, offset)
+	offset += len(b.Blobs) * 131072
+
+	// Field (0) 'KzgCommitments'
+	if size := len(b.KzgCommitments); size > 4096 {
+		err = ssz.ErrListTooBigFn("--.KzgCommitments", size, 4096)
+		return
+	}
+	for ii := 0; ii < len(b.KzgCommitments); ii++ {
+		if size := len(b.KzgCommitments[ii]); size != 48 {
+			err = ssz.ErrBytesLengthFn("--.KzgCommitments[ii]", size, 48)
+			return
+		}
+		dst = append(dst, b.KzgCommitments[ii]...)
+	}
+
+	// Field (1) 'Proofs'
+	if size := len(b.Proofs); size > 33554432 {
```

Comment:
Discussed offline. This seems incorrect. This check allows up to 1.6Gb of proof data

Reply:
Fixed in a9fb56df1d3c7caba2e44c4bcb64b6992056ed89.